force-app/main/default/classes/OSF_WeatherReader.cls

```diff
@@ -0,0 +1,92 @@
+/**********************************************************************
+ * @Name         : OSF_WeatherReader
+ * @Description  : Class to invoke openweathermap API in order to get the weather of the account city.
+ *
+ * @Created By   : Marta Demeny
+ * @Created Date : 18 jun. 2022
+ * @Modification Log: 
+ ********************************************************************
+ * Version		Developer		Date			Description
+ *___________________________________________________________________
+ *
+ *********************************************************************/
+public with sharing class OSF_WeatherReader {
+    private static final String WEATHER_URL; // = 'https://api.openweathermap.org/data/2.5/weather';
+    private static final String WEATHER_IMG_URL; // = 'https://openweathermap.org/img/wn/';
+    private static final String APPID; // = '9ab4d1340bf4d964ff4412a9068f7edc';
+    static {
+        //grab from custom setting values
+        Weather_Http_List_Setting__c settings = Weather_Http_List_Setting__c.getInstance('WeatherInfo');
+        System.debug('WeatherInfo: ' + settings);
+        WEATHER_URL = settings.Weather_API_URL__c;
+        APPID = settings.APPID__c;
+        WEATHER_IMG_URL = settings.Weather_IMG_URL__c;
+        System.debug('WEATHER_URL: ' + WEATHER_URL);
+        System.debug('WEATHER_IMG_URL: ' + WEATHER_IMG_URL);
+        System.debug('APPID: ' + APPID);  
```

Comment:
please clean up debugs from all the classes

Reply:
done